### PR TITLE
[Page header] Description max width 50 > 33.5rem

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Description max width 50 > 33.5rem

-----

image

-----

### CODE REVIEW

**Review by vvalentin-lucca — APPROVED** [review]

File: packages/scss/src/components/pageHeader/vars.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
 @mixin vars {
-	--components-pageHeader-description-max-width: 50rem;
+	--components-pageHeader-description-max-width: 33.5rem;
+	--components-pageHeader-description-max-width: 536px;
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/pageHeader/vars.scss:3]: on double les variables en rem et px ?

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2894/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18825/ui-diff)

File: packages/scss/src/components/pageHeader/vars.scss (hunk 1 above)
Comment (jeremie-lucca) [reply packages/scss/src/components/pageHeader/vars.scss]: Just a save mistake after comparing rem & px conversion 🫣

**Review by AlexiaAlarza — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2894/storybook

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18829/ui-diff)

**Pull request merged**